server/lists/adapters/__tests__/workstacks.spec.js

```diff
@@ -17,7 +17,7 @@ const mockFromStaticList = jest.fn((list) => {
     }
 });
 
-const todaysDate = new Date().toISOString().slice(0,10); // Returns todays date in format (yyyy-MM-dd)
+const todaysDate = new Date().toISOString().split('T')[0]; // Returns todays date in format (yyyy-MM-dd)
```

Comment:
```suggestion
onst todaysDate = new Date().toISOString().split('T')[0]; // Returns today's date in format (yyyy-MM-dd)
```

Reply:
Wonder if this should be in dateHelpers as well?

Reply:
This has been moved and tests added 👍🏼